addons/mail/models/mail_thread.py

```diff
@@ -121,15 +121,18 @@ def _compute_message_partner_ids(self):
     def _search_message_partner_ids(self, operator, operand):
         """Search function for message_follower_ids"""
         neg = ''
+        MailFollowers = self.env['mail.followers'].sudo()
+
         if operator in expression.NEGATIVE_TERM_OPERATORS:
             neg = 'not '
             operator = expression.TERM_OPERATORS_NEGATION[operator]
-        followers = self.env['mail.followers'].sudo()._search([
+        followers_query = MailFollowers._where_calc([
```

Comment:
It's not clear why this fixes the issue, shouldn't _search return a query object as per its docstring?